tests/debugging/run_module/target.py

```diff
@@ -0,0 +1,25 @@
+import threading
+from time import sleep
+
+from tests.debugging.mocking import TestDebugger
+
+
+def foo():
+    concealed = "secret"  # noqa
+    return
+
+
+def test():
+    sleep(0.5)
```

Comment:
why do we need to sleep here? To wait for `foo` to be registered?

Reply:
We give time to the module to finish loading, so that we are sure that the debugger watchdog triggered the instrumentation hooks and the target function is instrumented. When we call it, we then test that it generates a snapshot, as expected. Without the sleep, the thread would be racing with the module loading process.